platform/node/test/suite_implementation_recycle_map.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
```

Comment:
Instead of duplicating this file, can you please add the changes to the original suite_implementation and add a command line argument to toggle between the two variants? It looks like the majority of the code is the same.